package.json

```diff
@@ -28,20 +28,18 @@
     "test-viewer": "yarn unit-viewer && jest lighthouse-viewer/test/viewer-test-pptr.js",
     "test-lantern": "bash lighthouse-core/scripts/test-lantern.sh",
     "unit-core": "jest \"lighthouse-core/\"",
-    "unit-core:ci": "jest --runInBand --coverage --ci \"lighthouse-core/\"",
-    "unit-cli": "jest  --runInBand \"lighthouse-cli/\"",
-    "unit-cli:ci": "jest --runInBand --coverage --ci \"lighthouse-cli/\"",
-    "unit-viewer": "jest \"lighthouse-viewer/\"",
+    "unit-cli": "jest \"lighthouse-cli/\"",
+    "unit-viewer": "jest lighthouse-viewer/**/*-test.js",
```

Comment:
was this bringing in too many things the old way?

Reply:
> was this bringing in too many things the old way?

yeah, when we switched the viewer puppeteer test from mocha to jest, that meant the looser `jest lighthouse-viewer/` was running that test as a unit test which seems wrong and doesn't help coverage anyways because it runs against the built viewer in `dist/`, not the original source files.

Maybe the puppeteer tests should live somewhere else?